mapper/mapper.go

```diff
@@ -888,24 +892,17 @@ func filterCategoriesByTemplate(template string, categories []data.Category) []d
 	return categories
 }
 
-// mapBreadcrumb maps breadcrumb response from Zebedee to page model
-func mapBreadcrumb(page *model.SearchPage, bcs []zebedee.Breadcrumb) {
+// mapBreadcrumb appends parent bulletin and "Previous releases" to Zebedee breadcrumb response
```

Comment:
Not sure that's true.

Reply:
Removed the wrong godoc comment. Have fixed this